test/RadworksGovernor.t.sol

```diff
@@ -494,7 +508,178 @@ abstract contract Propose is ProposalTest {
   }
 }
 
-// Run the tests using the deployed Governor Bravo (future PR)
+abstract contract _Execute is ProposalTest {
+  function setUp() public virtual override(ProposalTest) {
+    ProposalTest.setUp();
+    _upgradeToBravoGovernor();
+  }
+
+  function _grantNewPauserViaGovernance(address _newPauser) internal {
+    (
+      address[] memory _targets,
+      uint256[] memory _values,
+      bytes[] memory _calldatas,
+      string memory _description
+    ) = _buildDripsGovernanceProposal(
+      "Grant Pauser role to an address",
+      _buildProposalData("grantPauser(address)", abi.encode(_newPauser))
+    );
+    _queueAndVoteAndExecuteProposalWithBravoGovernor(
+      _targets, _values, _calldatas, _description, FOR
+    );
+  }
+
+  function _revokePauserViaGovernance(address _newPauser) internal {
+    (
+      address[] memory _targets,
+      uint256[] memory _values,
+      bytes[] memory _calldatas,
+      string memory _description
+    ) = _buildDripsGovernanceProposal(
+      "Revoke Pauser role from an address",
+      _buildProposalData("revokePauser(address)", abi.encode(_newPauser))
+    );
+    _queueAndVoteAndExecuteProposalWithBravoGovernor(
+      _targets, _values, _calldatas, _description, FOR
+    );
+  }
+
+  function _proposeNewAdminViaGovernance(address _newAdmin) internal {
+    (
+      address[] memory _targets,
+      uint256[] memory _values,
+      bytes[] memory _calldatas,
+      string memory _description
+    ) = _buildDripsGovernanceProposal(
+      "Propose new Admin", _buildProposalData("proposeNewAdmin(address)", abi.encode(_newAdmin))
+    );
+    _queueAndVoteAndExecuteProposalWithBravoGovernor(
+      _targets, _values, _calldatas, _description, FOR
+    );
+  }
+
+  function _performDripsUpgradeViaGovernance(address _newImplementation) internal {
+    (
+      address[] memory _targets,
+      uint256[] memory _values,
+      bytes[] memory _calldatas,
+      string memory _description
+    ) = _buildDripsGovernanceProposal(
+      "Propose upgrade to new implementation",
+      _buildProposalData("upgradeTo(address)", abi.encode(_newImplementation))
+    );
+    _queueAndVoteAndExecuteProposalWithBravoGovernor(
+      _targets, _values, _calldatas, _description, FOR
+    );
+  }
+
+  function testFuzz_grantPauserOnDrips(address _newPauser) public {
```

Comment:
grant should be capitalized and the testing should follow bullet 6 in the [Foundry Best Practices](https://book.getfoundry.sh/tutorials/best-practices?highlight=best#tests). It would be good to double check the names in the other tests.

Reply:
So maybe something like `testFuzz_CanGrantPauserOnDrips`

Reply:
Done .. for a few test names: https://github.com/ScopeLift/radworks-governor-upgrade/pull/22/commits/d2f8877b757dbce3be2daa4288713bb0bf4f64ed